imap_processing/tests/ialirt/unit/test_decom_ialirt_hit.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from imap_processing import imap_module_directory
+from imap_processing.decom import decom_packets
+from imap_processing.ialirt.l0.decom_ialirt import generate_xarray
+
+
+@pytest.fixture(scope="session")
+def xtce_hit_path():
+    """Returns the xtce auxiliary directory."""
+    return imap_module_directory / "ialirt" / "packet_definitions" / "ialirt_hit.xml"
+
+
+@pytest.fixture(scope="session")
+def binary_packet_path():
+    """Returns the xtce auxiliary directory."""
+    return (
+        imap_module_directory
+        / "tests"
+        / "ialirt"
+        / "test_data"
+        / "l0"
+        / "hit_ialirt_sample.ccsds"
+    )
+
+
+@pytest.fixture(scope="session")
+def hit_test_data():
+    """Returns the xtce auxiliary directory."""
+    header_path = (
+        imap_module_directory
+        / "tests"
+        / "ialirt"
+        / "test_data"
+        / "l0"
+        / "hit_ialirt_sample_header.csv"
+    )
+    header = list(pd.read_csv(header_path).columns)
+
+    data_path = (
+        imap_module_directory
+        / "tests"
+        / "ialirt"
+        / "test_data"
+        / "l0"
+        / "hit_ialirt_sample.csv"
+    )
+    data = pd.read_csv(
+        data_path, header=None, names=header, na_values=[" ", ""]
+    ).astype("float")
```

Comment:
* Can you manually add the header onto the sample data? This is a bit confusing to read in two different files, grabbing the columns from one dataset and adding it to another.
* I don't think we want "float", these are integer values being read in. pandas read_csv should auto-infer the types of columns for you, so can we just let that do its thing?